packages/utils/src/feature-flags/utils/flag-router.ts

```diff
@@ -1,17 +1,63 @@
 import { FeatureFlagTypes } from "@medusajs/types"
 
 export class FlagRouter implements FeatureFlagTypes.IFlagRouter {
-  private readonly flags: Record<string, boolean> = {}
```

Comment:
todo: same comment as in medusa package

Reply:
There's no comment to this in the Medusa package.

Reply:
yes there is, see previous comments

Reply:
Can you link it? I can't see it :)

Reply:
https://github.com/medusajs/medusa/pull/4701#discussion_r1285514179
https://github.com/medusajs/medusa/pull/4701#discussion_r1285515404
https://github.com/medusajs/medusa/pull/4701#discussion_r1285522556

those are in both places for now right?